### PR TITLE
JSON importer

### PR DESCRIPTION
**Pull request by ahmetabdi**

### CODE REVIEW

**Review by steview — CHANGES_REQUESTED** [review]
Only one comment. Can we namespace the import please MTGDB::Importers::JSONImporter. WDYT?

**Comment by ahmetabdi** [comment]
Sounds good to me I'll make that change

File: spec/lib/mtg_json_importer_spec.rb [hunk 1]
```diff
@@ -0,0 +1,150 @@
+require 'rails_helper'
+require 'mtg_json_importer'
+
+RSpec.describe MtgJsonImporter do
+  describe '.run' do
+    context 'given json for a set' do
```
Comment (steview) [inline spec/lib/mtg_json_importer_spec.rb]: I'm not sure we need contexts for this spec, I think it should be one scenario using the JSON data _with_ the card data. Maybe use describe blocks to neatly divide the set and card based specs. WDYT?

**Review by steview — CHANGES_REQUESTED** [review]

**Comment by ahmetabdi** [comment]
Any idea why this doesn't work @steview 

Class:
`class MTGDB::Importers::JSONImporter`

Output:
`uninitialized constant MTGDB`

**Comment by steview** [comment]
@ahmetabdi Is the directory structure correct? Are you including it correctly in the init?

**Review by steview — APPROVED** [review]

**Pull request merged**